### PR TITLE
Career page view

### PR DESCRIPTION
**Pull request by Meghana-12**
Related to issue #396
View the page on route `/careersPage`

CareersPage

### CODE REVIEW

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Footer/Components/index.tsx [hunk 1]
```diff
@@ -109,11 +109,7 @@ const PureFooter: React.FunctionComponent<Props> = ({
           </li>
           <li>
             {' '}
-            <a
-              target="_blank"
-              rel="noopener noreferrer"
-              href="https://medium.com/swlh/ignitus-connecting-students-with-opportunities-61f08e19861c"
-            >
+            <a target="_blank" rel="noopener noreferrer" href="/careerPage">
```
Comment (divyanshu-rawat) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Footer/Components/index.tsx:112]: Why have you added a career page link to Blog? nevertheless and the link should be `careersPage` not `careerPage`.

File: src/ignitus-CareersPage/Components/index.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+              </S.Paragraph>
+            </S.Wrapper>
+          ))}
+        </S.Wrapper>
+        <S.Wrapper flex="1">
+          <S.Img src={developer} alt="DeveloperImg" />
+        </S.Wrapper>
+      </S.FlexRow>
+
+      <S.FlexRow>
```
Comment (divyanshu-rawat) [inline src/ignitus-CareersPage/Components/index.tsx:105]: Can you create a component for the same? as FlexRow is repeating for every job.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Footer/Components/index.tsx (hunk 1 above)
Comment (divyanshu-rawat) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Footer/Components/index.tsx:112]: I will tc of that.

**Comment by divyanshu-rawat** [comment]
@Meghana-12 you can pick up another issue I will take care of remaining refactoring :)

File: src/ignitus-CareersPage/styles.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  flex: 1 1 30%;
+  padding: 1rem;
+  display: flex;
+  flex-direction: row;
+  justify-contents: flex-start;
+`;
+
+export const FlexRow = styled.div`
+  display: flex;
+  flex-direction: ${(props: StyleTypes) => props.direction};
```
[the remark this answers is not in the document]
Comment (Meghana-12) [reply src/ignitus-CareersPage/styles.ts]: Okay

File: src/ignitus-CareersPage/Components/index.tsx [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+      </S.Heading>
+      <br />
+      <br />
+      <RoundedButton size="large" category="secondary">
+        {' '}
+        We&apos;re Hiring{' '}
+      </RoundedButton>
+    </S.TopSection>
+
+    <WhySection title="Why Ignitus?" />
```
[the remark this answers is not in the document]
Comment (Meghana-12) [reply src/ignitus-CareersPage/Components/index.tsx]: Sure, I'll remember that.

File: src/ignitus-CareersPage/styles.ts [hunk 5]
```diff
@@ -0,0 +1,114 @@
+import styled from '@emotion/styled';
+import * as C from '../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/colors';
```
[the remark this answers is not in the document]
Comment (Meghana-12) [reply src/ignitus-CareersPage/styles.ts]: okay

**Pull request merged**